README.md

```diff
@@ -31,8 +33,7 @@ But it is strongly recommended to only use [GitHub Actions "Release to Maven Cen
 
 ## License
 
-License is Apache-2. Each source file must include the Apache-2 header (build will fail otherwise).
-To update source files with the proper header, simply execute the below command:
+License is GNU General Lesser Public License (LGPL) version 3.0. Each source file includes the LGPL-3 header (build will fail otherwise). To update source files with the proper header, simply execute the below command:
```

Comment:
If i'm not mistaken (to be confirmed), we'll use Apache-2 as a license in this module. LGPL V3 is only used when the module uses another protected code from another organization.

Reply:
Please use Apache-2 as a license.